std_msgs/Header header

int32 gamemode
int32 GAMEMODE_CHECK_BARREL = 1
int32 GAMEMODE_CHECK_H = 2
int32 GAMEMODE_AUTOLAND = 3
int32 GAMEMODE_SURVEY = 4
int32 GAMEMODE_OTHER = 0

# string[] GAMEMODE_STR
# GAMEMODE_STR[0] = "GAMEMODE_OTHER"
# GAMEMODE_STR[1] = "GAMEMODE_CHECK_BARREL"
# GAMEMODE_STR[2] = "GAMEMODE_CHECK_H" 
# GAMEMODE_STR[3] = "GAMEMODE_AUTOLAND" 
# GAMEMODE_STR[4] = "GAMEMODE_SURVEY"
